config/singleFrameDriver.py

```diff
@@ -0,0 +1,9 @@
+"""
+Subaru-specific overrides for SingleFrameDriverTask (applied before SuprimeCam- and HSC-specific overrides).
+"""
+import os.path
+
+from lsst.utils import getPackageDir
+
+config.processCcd.load(os.path.join(getPackageDir("obs_subaru"), "config", "processCcd.py"))
+config.processCcd.load(os.path.join(getPackageDir("obs_subaru"), "config", "hsc", "processCcd.py"))
```

Comment:
Right, there needs to be a `config/hsc/singleFrameDriver.py` to pull the HSC-specific overrides.